_data/footer.yml

```diff
@@ -7,4 +7,7 @@
     link: https://github.com/openfreeenergy
     new_window: True
     social_icon: GitHub  # we'll need to get this from another source
-
+  - name: Publications
+    link: https://zenodo.org/communities/openfreeenergy
+    new_window: True
+    social_icon: Zenodo # Not sure if this works
```

Comment:
doesn't exist yet, but you can add the icon SVG to `_includes/social-icon.html`. A Zenodo icon is [available from simpleicons.org](https://simpleicons.org/?q=zenodo).

Reply:
added :)